.vscode/settings.json

```diff
@@ -19,7 +19,7 @@
     "editor.defaultFormatter": "ms-python.black-formatter"
   },
   "editor.tabSize": 4,
-  "editor.formatOnSave": true,
+  "editor.formatOnSave": false,
```

Comment:
We need to stop changing this back and forth 😅 is the auto save causing trouble? Works fine on our end

Reply:
For me, the issue is that BUILD files are python files but they are not currently formated as our other python files.
So each time I edit a BUILD file, the formatting is all changed.

Have you disabled the formating of BUILD files?